cmd/brew-test-bot.rb

```diff
@@ -977,7 +980,10 @@ def test_ci_upload(tap)
 
     if (pr = ENV["UPSTREAM_PULL_REQUEST"])
       pull_pr = "https://github.com/#{tap.user}/homebrew-#{tap.repo}/pull/#{pr}"
-      safe_system "brew", "pull", "--clean", pull_pr
+      args = []
+      args << "--tap=#{@tap}" unless @tap.nil?
+      args << pull_pr
+      test "brew", "pull", "--clean", *args
```

Comment:
`Linuxbrew/science` is a fork of `Homebrew/science` used for building bottles without cluttering up the issue tracker of `Homebrew/science`. The URL of the repo is https://github.com/Linuxbrew/homebrew-science but the tap is `Homebrew/science`, as if you had done `brew tap homebrew/science https://github.com/Linuxbrew/homebrew-science`. It's tested using `brew test-bot --tap=homebrew/science`, and `test-bot` needs to pass the `--tap` parameter to `brew pull`, for example with `brew pull --tap=homebrew/science https://github.com/Linuxbrew/homebrew-science/pull/342`. I also use this arrangement to build bottles for `Homebrew/dupes`. That's the situation. Is what I've done the best way to fix this issue?